components/massmint/UploadPicture/UploadPicture.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="collpase-massmint">
+    <NeoCollapse>
+      <template #title> {{ $t('massmint.uploadPics') }}</template>
+      <template #content>
+        <DropUpload
+          preview
+          :label="$t('mint.collection.massmintDrop')"
+          accept="image/*, audio/*, .zip"
+          @input="selectedFile" />
+      </template>
+    </NeoCollapse>
+    <NeoCollapse>
+      <template #title> {{ $t('massmint.uploadDesc') }} </template>
+      <template #content> custom content </template>
+    </NeoCollapse>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { NeoCollapse, NeoCollapseStack } from '@kodadot1/brick'
+
+const DropUpload = defineAsyncComponent(
+  () => import('@/components/shared/upload/DropUpload.vue')
+)
+
+const selectedFile = (file) => {
+  console.log('selected file', file)
+}
```

Comment:
isnt the disabled state depended on whether the collection is chosen or not in the previous step? I dont get how selectedFile event is related to disabled?

Reply:
The selected file will toggle the disabled state of the next collapse (description)

It would be great if you also use the collection chosen/not chosen to toggle disabled for upload images collapse as well